sbom.cdx.json

```diff
@@ -0,0 +1,39 @@
+{
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "version": 1,
+  "components": [
+    {
+      "type": "library",
+      "bom-ref": "pkg:github/provider-corner/libprov@@VCS_TAG@",
+      "cpe": "cpe:2.3:a:provider-corner:libprov:@VCS_TAG@:*:*:*:*:*:*:*",
+      "name": "libprov",
+      "version": "@VCS_VERSION@",
+      "description": "Small library of helpers for OpenSSL 3 providers",
+      "authors": [
+        {
+          "name": "@VCS_SBOM_AUTHORS@"
+        }
+      ],
```

Comment:
If I understand your explanation of `@VCS_SBOM_AUTHORS@` correctly, this should be `metadata.authors`, not `components.[].authors`, and `components.[].authors` should be the set of component authors, not the SBOM authors.  It would be nice if there was a `@VCS_AUTHORS` for which the result of `git shortlog -n -s` was used.

Refs: 
https://cyclonedx.org/docs/1.6/json/#metadata_authors
https://cyclonedx.org/docs/1.6/json/#components_items_authors

Reply:
I think you're dead right about the `metadata.authors` change. I've got 5 pull requests to modify! :)

>  It would be nice if there was a @VCS_AUTHORS@

Great idea: https://github.com/hughsie/python-uswid/pull/66